### ISSUE TITLE
PWA option in chrome not available 

### ISSUE THREAD

**Issue by spencerwongfeilong** [Kerwin1202/chatgpt-web#45]
Hi

I am under the impression PWA is available. However in android chrome I can't install the app. Is there something I missed?

**Comment by luckywangxi** [comment]
Yes, PWA is unavailable, you may restore the changes https://github.com/Chanzhaoyu/chatgpt-web/commit/d2a22a3ee5d59affa7f3ca0a85491b4f6e87fc44
